Makefile

```diff
@@ -6,12 +6,7 @@ PATH_WITH_TOOLS="`pwd`/$(TOOL_BIN):`pwd`/node_modules/.bin:${PATH}"
 
 GIT_REVISION = $(shell git rev-parse HEAD | tr -d '\n')
 TAG_VERSION?=$(shell etc/tag_version.sh)
-LDFLAGS = -ldflags "$(shell etc/set_plt.sh) -X 'go.viam.com/rdk/config.Version=${TAG_VERSION}' -X 'go.viam.com/rdk/config.GitRevision=${GIT_REVISION}'"
-
-# build tags are not currently needed, but leaving placeholders in case future requirements need a tag again
-# BUILD_TAGS=dynamic
-# GO_BUILD_TAGS = -tags $(BUILD_TAGS)
-# LINT_BUILD_TAGS = --build-tags $(BUILD_TAGS)
+LDFLAGS = -ldflags "-s -w -extld="$(shell pwd)/etc/ld_wrapper.sh" -X 'go.viam.com/rdk/config.Version=${TAG_VERSION}' -X 'go.viam.com/rdk/config.GitRevision=${GIT_REVISION}'"
```

Comment:
we now lose out on the version tagging for the normal server here on macos. maybe we should split these flags up?